airbyte-integrations/connectors/source-chargebee/source_chargebee/schemas/addon.json

```diff
@@ -153,7 +135,8 @@
       "maxLength": 50
     },
     "meta_data": {
-      "type": ["object", "null"]
+      "type": ["object", "null"],
+      "properties": {}
```

Comment:
Why specify this if it is empty?

Reply:
Because I don't know properties of `meta_data` field. Stripe connector, for example, also has empty `properties` field if properties are not known, as well as few other connectors.

![image](https://user-images.githubusercontent.com/19872253/127525795-d2cfc9ab-fd27-4ad2-9b54-462cc3896174.png)